### DISCUSSION TITLE
Coinbase Advanced Trade

### DISCUSSION THREAD

**Discussion by LucidDion** [bchavez/Coinbase.Pro#64] [Q&A]
Are there any plans for the new API support, Coinbase Advanced Trade?

**Comment by HawksterVJ** [comment]
Hello. I am interested in this. There is a package called cbpro on here that is used with the API for CoinbasePro. I am really hoping that one very similar will be written for Coinbase Advanced Trading. Can anyone comment on this? Any help is greatly appreciated.

Thanks
